marklogic-data-hub-central/ui/src/components/detail-page-non-entity/detail-page-non-entity.tsx

```diff
@@ -173,6 +174,34 @@ const DetailPageNonEntity = (props) => {
     }
   };
 
+  const download = async () => {
+    try {
+      const response = await axios({
```

Comment:
Any axios call should be moved out to the /api folder (may be create a new file for explorer there ?) and then the function can be imported here, to follow similar code pattern as other components.